src/Container/WhoopsFactory.php

```diff
@@ -75,11 +75,11 @@ private function registerJsonHandler(Whoops $whoops, $config)
 
         $handler = new JsonResponseHandler();
 
-        if (isset($config['json_exceptions']['show_trace'])) {
+        if (isset($config['json_exceptions']['show_trace']) && true === $config['json_exceptions']['show_trace']) {
```

Comment:
`! empty` is functionally equivalent here, more succinct, and more in line with other similar checks against configuration flags we're doing elsewhere.